tests/unit/amo/components/TestScreenShots.js

```diff
@@ -49,14 +49,15 @@ describe('<ScreenShots />', () => {
   });
 
   it('renders custom thumbnail', () => {
-    const item = { src: 'https://foo.com/img.png' };
+    const item = { src: 'https://foo.com/img.png', title: 'test title' };
     const thumbnail = shallow(thumbnailContent(item));
 
     expect(thumbnail.type()).toEqual('img');
     expect(thumbnail.prop('src')).toEqual('https://foo.com/img.png');
     expect(thumbnail.prop('height')).toEqual(HEIGHT);
     expect(thumbnail.prop('width')).toEqual(WIDTH);
     expect(thumbnail.prop('alt')).toEqual('');
+    expect(thumbnail.prop('title')).toEqual('test title');
```

Comment:
I know you were just following the pattern but for future reference we prefer this check now:

```js
expect(thumbnail).toHaveProp('title', 'test title');
```

This produces a better failure message.